go/base/context.go

```diff
@@ -290,6 +290,21 @@ func NewMigrationContext() *MigrationContext {
 	}
 }
 
+func (this *MigrationContext) SetConnectionConfig(storageEngine string) error {
+	var transactionIsolation string
+	switch storageEngine {
+	case "rocksdb":
+		transactionIsolation = "READ-COMMITTED"
+	case "innodb":
```

Comment:
Minor nit: the `case "innodb":` could be removed as `default:` does the same thing